### PR TITLE
#2844 cleanup sormas2sormas config properties

### PR DESCRIPTION
**Pull request by leventegal-she**

### CODE REVIEW

File: sormas-base/setup/sormas.properties [hunk 1]
```diff
@@ -58,19 +58,30 @@ app.url=
 # default: "/opt/sormas/custom/"
 #custom.path=/opt/sormas/custom/
 
+# Key data for the SORMAS to SORMAS feature
```
Comment (HolgerReiseVSys) [inline sormas-base/setup/sormas.properties:61]: Maybe "Configuration for the SORMAS to SORMAS feature"?
Or should this go below `#sormas2sormas.path=`?

File: sormas-base/setup/sormas.properties [hunk 2]
```diff
@@ -58,19 +58,30 @@ app.url=
...
+
 # Path where SORMAS to SORMAS related certificates and files are stored
 # default: "/opt/sormas/sormas2sormas/"
-#sormas2sormas.path=/opt/sormas/sormas2sormas/
-
-# Key data for the generated SORMAS to SORMAS certificate"
 #sormas2sormas.path=
+
+# The name of the file containing server access data: organization id, organization name, host name of the server, REST user password
+# default: "/opt/sormas/sormas2sormas/{host name}-server-access-data.csv"
```
Comment (HolgerReiseVSys) [inline sormas-base/setup/sormas.properties:68]: Technically these are not default values, just the values added by the script, right?
So we probably just write `#sormas2sormas.serverAccessDataFileName=/opt/sormas/sormas2sormas/{host name}-server-access-data.csv`

**Review by HolgerReiseVSys — CHANGES_REQUESTED** [review]

File: sormas-base/setup/sormas.properties (hunk 1 above)
Comment (leventegal-she) [reply sormas-base/setup/sormas.properties:61]: I wanted this to be a title for all the s2s related keys so not below  sormas2sormas.path but mark somehow that the following keys belong to sormas 2 sormas.

Do you have any suggestion how to "group" these keys?

Comment (HolgerReiseVSys) [reply sormas-base/setup/sormas.properties:61]: `Configuration for the SORMAS to SORMAS feature`?
Maybe then we won't need `# SORMAS to SORMAS truststore data` either?

File: sormas-base/setup/sormas.properties [hunk 3]
```diff
@@ -58,19 +58,24 @@ app.url=
...
-# Key data for the generated SORMAS to SORMAS certificate"
-#sormas2sormas.path=
-#sormas2sormas.serverAccessDataFileName=
-#sormas2sormas.hostName=
-#sormas2sormas.keystoreName=
+# The name of the file containing server access data: organization id, organization name, host name of the server, REST user password
+#sormas2sormas.serverAccessDataFileName="{host name}-server-access-data.csv"
+
+# The name of the generated key store file
+#sormas2sormas.keystoreName="{host name}.sormas2sormas.keystore.p12"
```
Comment (HolgerReiseVSys) [inline sormas-base/setup/sormas.properties:70]: remove quotes?

File: sormas-base/setup/sormas.properties [hunk 4]
```diff
@@ -58,19 +58,24 @@ app.url=
...
+
+# Password of the key store
 #sormas2sormas.keystorePass=
 
-# SORMAS to SORMAS truststore data"
-#sormas2sormas.truststoreName=
+# The name of the trust store file
+#sormas2sormas.truststoreName="sormas2sormas.truststore.p12"
```
Comment (HolgerReiseVSys) [inline sormas-base/setup/sormas.properties:76]: remove quotes?

**Review by HolgerReiseVSys — CHANGES_REQUESTED** [review]

**Review by HolgerReiseVSys — APPROVED** [review]

**Pull request merged**